Toolset/home.livecodescript

```diff
@@ -1207,6 +1208,28 @@ command revInternal__InitialiseLibraries
    try
       revIDEDeployInitialize
    end try
+   
+   -- Load libraries in the Ext externals bundle
+   if there is a folder revEnvironmentExtPath() then
+      set the folder to revEnvironmentExtPath()
+      repeat for each line tExternal in the folders
+         if tExternal is empty or tExternal is "." or tExternal is ".." then
+            next repeat
+         end if
+         set the folder to revEnvironmentExtPath() & "/" & tExternal
+         
+         local tFiles
+         put the files into tFiles
+         filter tFiles with "*lib*.livecode*"
+         
+         repeat for each line tFile in tFiles
+            if there is a stack tFile then
+               start using stack tFile
+            end if
```

Comment:
Yes that's one of the things I need to change along with the initialisation library that should remove a stack from memory if it doesn't handle revLoadLibrary